### PR TITLE
[MIRROR] Adds a bunch of new clown names to the .txt

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/43113
----------------------------

## About The Pull Request
Exactly what it says on the tin. Adds a bunch of new clown names.

## Why It's Good For The Game
Because the current list is too short. I see too many of the same clown names, and want to fix that. With more clown names.

## Changelog
:cl:
tweak: Added a big heap of fancy new clown names! Be slightly less generic!
/:cl:

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
Nice.

File: strings/names/clown.txt [hunk 1]
```diff
@@ -27,11 +27,97 @@ Razzle Dazzle
...
+Bimbim
+Greedo Shotfurst
+Tree Talker
+Circe The Great
+Mister Redherring
+Polite Pablo
+Rude Robert
+Doctor Greenthumb
+Sandwich Sam
+Red Dead Redemption Two
```
Comment (Jujumatic) [inline strings/names/clown.txt:100]: :megathink:

**Comment by TheMeatGod** [comment]
Lmao

File: strings/names/clown.txt (hunk 1 above)
Comment (plapatin) [reply strings/names/clown.txt:100]: https://i.imgur.com/SMrqMz9.png

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**